<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConvolve</class>
 <widget class="QDialog" name="DialogConvolve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>nsxqt Peak Finder</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>0</y>
     <width>481</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="3">
     <widget class="QSpinBox" name="maxCompBox">
      <property name="maximum">
       <number>2000000000</number>
      </property>
      <property name="value">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Threshold type</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="maxCompLabel">
      <property name="text">
       <string>Max. componenents</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QDoubleSpinBox" name="integrationConfidenceSpinBox">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
      <property name="value">
       <double>0.997000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QComboBox" name="thresholdComboBox">
      <item>
       <property name="text">
        <string>Relative to background</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Absolute</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Filter parameters</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="minCompLabel">
      <property name="text">
       <string>Min. components</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="confidenceLabel">
      <property name="text">
       <string>Integration confidence</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="filterLabel">
      <property name="text">
       <string>Filter type</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="thresholdLabel">
      <property name="text">
       <string>Threshold value</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QSpinBox" name="minCompBox">
      <property name="maximum">
       <number>2000000000</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QPushButton" name="previewButton">
      <property name="text">
       <string>Generate Preview</string>
      </property>
     </widget>
    </item>
    <item row="12" column="3">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QCheckBox" name="thresholdCheckBox">
      <property name="text">
       <string>apply threshold to preview</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QDoubleSpinBox" name="thresholdSpinBox">
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QTreeView" name="treeView"/>
    </item>
    <item row="7" column="3">
     <widget class="QComboBox" name="filterComboBox">
      <property name="insertPolicy">
       <enum>QComboBox::InsertAlphabetically</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QDoubleSpinBox" name="blobConfidenceSpinBox">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
      <property name="value">
       <double>0.660000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="confidenceLabel_2">
      <property name="text">
       <string>Peak search confidence</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogConvolve</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogConvolve</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
